wls-auth-service/src/main/resources/db/migrations/h2/V10_0_addAdminPermissionsGetKonfigurierteWahltage.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO Permission
+VALUES ('00000000-0000-0009-0000-000000000001', 'Admin_BUSINESSACTION_GetKonfigurierteWahltage');
```

Comment:
Ich glaube diese Permission-Id ist schon im V9_0 vergeben.
```suggestion
VALUES ('00000000-0000-0010-0000-000000000001', 'Admin_BUSINESSACTION_GetKonfigurierteWahltage');
```